packages/block-editor/src/components/block-list/index.native.js

```diff
@@ -162,12 +162,13 @@ export class BlockList extends Component {
 		return this.extraData;
 	}
 
-	getCellRendererComponent( { children, item } ) {
+	getCellRendererComponent( { children, item, onLayout } ) {
 		const { rootClientId } = this.props;
 		return (
 			<BlockListItemCell
 				children={ children }
 				clientId={ item }
+				listOnLayout={ onLayout }
```

Comment:
I wonder if this name could be different than the current one, maybe something like `internalListOnLayout`?

Reply:
I'm thinking that we could simply name it `onLayout` as we'd do for other components, wdyt?